sageintacctsdk/apis/constants.py

```
@@ -756,6 +756,61 @@
         "CREDITAMOUNT",
         "AMOUNT",
     ],
+    "GLENTRY": [
+    "RECORDNO",
```

Flake8 (7.2.0)

[error] continuation line missing indentation or outdented

(E122)